src/main/java/newbank/client/TestClient.java

```diff
@@ -0,0 +1,20 @@
+package newbank.client;
+
+import java.io.IOException;
+import java.io.Reader;
+
+public class TestClient extends ExampleClient {
+
+  public TestClient(String ip, int port, Reader reader)
+      throws IOException, IllegalThreadStateException, ConfigurationException {
+    super(ip, port, reader);
+  }
+
+  protected void displayPrompt() {
+    // NOT NEEDED FOR TEST CLIENT
+  }
+  
+  protected void processResponse(final String response) {
+	  display.writeLine(response);
```

Comment:
The formatting seems inconsistent in this file.

Reply:
Are you sure? Let me double check...

Reply:
Good catch, re-applied the formatting to that file.

Kindest Regards,
Jules